modules/ROOT/pages/dataweave-indexed-readers.adoc

```diff
@@ -0,0 +1,51 @@
+= Indexed Readers in DataWeave
+
+DataWeave can read input data (such as XML, JSON, and CSV) in an indexed strategy to avoid loading the whole document in memory while still allowing random access just like in the in-memory strategy. Use indexed readers when the input is larger than a certain configurable threshold or if the `indexedReader` setting is set to `true`.
+
+To manage files that don’t fit in memory, write the files to a temporary file in disk and index the document content beforehand, which also takes time and disk space.
+
+Indexed readers can process input files up to 20GBs. For bigger files use the streaming mode, which is faster. Despite this mode being more limited in functionality, there’s no maximum input size limitation for streaming readers.
```

Comment:
What is the purpose of this paragraph? 
It clearly states a size limitation for indexed readers...which are used to overcome the size of big incoming files.
Why do we then mention streaming mode (as opposed to what other mode? Indexed) for even bigger files.
And how is streaming mode limited if it is faster and has no max input size?

Reply:
Maybe it is my lack of tech knowledge about reading input data, but I am not understanding the differences among....

- "normal" reading of input data
-  indexed readers for large input data (time consuming, space consuming)
-  streaming reading for even larger input data (faster, no memory limit)

The focus should be on defining what indexed readers are, compared to other reading methods (?), and how and why your audience can use them.